themes/default/content/product/_index.md

```diff
@@ -12,99 +12,203 @@ subheading: |
-                    encapsulate your cloud architectures and best practices.
+                public class App {
+                    public static void main(String[] args) {
+                        Pulumi.run(App::stack);
+                    }
 
-        - description: Deploy cloud infrastructure and applications together
+                    private static void stack(Context ctx) {
+                    final var cluster = new Cluster("eks-cluster");
+                    ctx.export("kubeconfig", cluster.kubeconfig());
+                  }
+                }
+            - title: Pulumi.yaml
+              language: yaml
+              code: |
+                resources:
+                  eks-cluster:
+                    type: eks:Cluster
+                outputs:
+                  kubeconfig: ${cluster.kubeconfig}
+          button:
+            text: "Learn more about Pulumi Packages"
+            link: "/product/packages/"
           features:
-              - title: Multiple deployment options
-                icon: rocketship
-                icon_color: salmon
+              - title: Native cloud providers
                 description: |
-                    Deploy infrastructure interactively with a CLI, programmatically with Pulumi’s
-                    [Automation API](/automation/), or through your [CI/CD process](/docs/guides/continuous-delivery/).
-
-              - title: Deployments as Code
-                icon: gear
-                icon_color: salmon
+                    Full API coverage for AWS, Azure, Google Cloud, and Kubernetes with same-day updates.
+              - title: Crosswalk for AWS
                 description: |
-                    Run deployments from your application code at runtime with [Automation API](/automation/). Create infrastructure APIs, custom platforms, and CLIs.
-
-              - title: Preview and test changes
-                icon: eye
-                icon_color: salmon
+                    Adopt well-architected best practices for your infrastructure easily with the [Crosswalk library](/docs/guides/crosswalk/aws/).
+              - title: Cloud Native support
                 description: |
-                    Test and validate infrastructure with standard [unit test frameworks](/docs/guides/testing/#unit-testing) and
-                    [integration tests](/docs/guides/testing/integration/). Preview changes before deploying.
-
-        - description: Manage cloud applications with visibility and controls
+                    Use a single workflow to manage cloud native infrastructure and [Kubernetes](/kubernetes/).
+        
+        - title: "Deliver infrastructure through software delivery pipelines"
+          sub_title: "CI/CD Integrations"
+          description: |
+            Version, review, test, and deploy infrastructure code through the same tools and processes used for your application code.
+          image: "/images/product/pulumi-cicd.png"
```

Comment:
the whitespace in the graphic is causing it to look like there is more whitespace between image & text here than in the other sections

Reply:
not sure how to get around this as this is a template that is abstracted to repeat itself for all the key features. I can maybe photoshop the image a bit to crop out the whitespace border around the logos.

Reply:
@gthuang I photoshopped out some of the surrounding whitespace in the image. let me know what you think.